txselector/txselector.go

```diff
@@ -762,17 +760,15 @@ func splitL2ForgableAndNonForgable(tp *txprocessor.TxProcessor,
 	var l2TxsForgable, l2TxsNonForgable []common.PoolL2Tx
 	for i := 0; i < len(l2Txs); i++ {
 		accSender, err := tp.StateDB().GetAccount(l2Txs[i].FromIdx)
+		l2Txs[i].Info = fmt.Sprintf("Tx not selected because failed to get account with idx %d", l2Txs[i].FromIdx)
```

Comment:
You must set this message inside the error condition below

Reply:
Maybe it is better to add the parameter name that was wrong (`FromIdx`) into the message:
```suggestion
		l2Txs[i].Info = fmt.Sprintf("Invalid transaction, FromIdx account not found %d", l2Txs[i].FromIdx)
```